### DISCUSSION TITLE
disable auto reverse of Menu

### DISCUSSION THREAD

**Discussion by felixfeng33** [ariakit/ariakit#3962] [Questions] [answered]
Hi, this is an excellent library  I do not want the automatic reversed feature when there is insufficient space. I would like to disable this feature and handle it myself by calculating and scrolling down.However, I couldn’t find the related API to disable this feature. I noticed similar APIs in radix and floating-ui. Did I miss something?​Thanks in advance.

**Comment by diegohaz** [comment, marked as answer]
It's hard to understand what you mean without a reproduction or video. Is it related to [`flip`](https://ariakit.org/reference/menu#flip) in some way?
